aksel.nav.no/website/pages/eksempler/button/neutral.tsx

```diff
@@ -45,4 +27,5 @@ export const Demo = {
 
 export const args = {
   index: 1,
+  desc: "For nøytrale handlinger som verken er positive eller negative, kan du bruke en 'nøytral' knapp.",
```

Comment:
I think it's more about attention. From the docs: "Disse knappene kan du bruke til handlinger som ikke trenger så mye oppmerksomhet."


```suggestion
  desc: "For handlinger som ikke trenger så mye oppmerksomhet, kan du bruke en 'nøytral' knapp.",
```